agent/runner/jobs/pbm_helpers.go

```diff
@@ -30,17 +31,31 @@ import (
 )
 
 const (
-	// How many times check if backup/restore operation was started
-	maxBackupChecks  = 10
-	maxRestoreChecks = 10
+	// How many times check if backup operation was started
+	maxBackupChecks = 10
+
+	// how many times to check if a restore operation has completed.
```

Comment:
Nope, we can't do that. We don't know how much time this operation will require time. We should wait until we get some status.